### ISSUE TITLE
Workaround when Notification access is not available

### ISSUE THREAD

**Issue by Primokorn** [UriahShaulMandel/BaldPhone#21]
**Is your feature request related to a problem? Please describe.**
There's a dot in the Messages card when receiving a SMS. This is not possible on devices running Android Go (feature limited by Google).

**Describe the solution you'd like**
I'd like to know if a workaround is possible.

**Describe alternatives you've considered**
My grandmother used to having this [big launcher](https://play.google.com/store/apps/details?id=name.kunes.android.launcher.demo). I removed it to install Baldphone.
I don't know how it worked but the card of Messages was red instead of yellow when she received messages. What could be the workaround used by this alternative launcher to know when there's an unread message?
For the time being I will display a popup through Macrodroid.

Thanks.

**Comment by UriahShaulMandel** [comment]
Hi, I need to check into the subject.
I'll post an update here when I'll find a solution. 
Adding "Help Wanted" so others can help

**Comment by Primokorn** [comment]
What about this:
- Display the red dot when a new SMS is received
- Remove the red dot once the default SMS app is opened

**Comment by UriahShaulMandel** [comment]
The problem is that I don't want to ask for another permission, BaldPhone asks for a lot of peemissions right now already, and it may scare elderly.

**Comment by Primokorn** [comment]
They don't know what permissions are :-)

**Comment by UriahShaulMandel** [comment]
Yes, and when I tested BaldPhone with users, some of them were really afraid and decided not to do it, so I'm trying to ask only for must-have permissions.

<del>With that said, I wanted for a long time for BaldPhone to replace the SMS interface too. It's not a simple task and may take a lot of time, but I hope soon I'll have a few days to create it.</del>

I checked online and apparently, it's much more difficult than what I have thought. Can't say for sure it'll be soon

> For the time being I will display a popup through Macrodroid

I hope it's fine, I can't find any simple solution at the meantime... hope you understand
